functions/_fzf_search_directory.fish

```diff
@@ -16,10 +16,10 @@ function _fzf_search_directory --description "Search the current directory. Repl
         set --append fd_opts --base-directory=$unescaped_exp_token
         # use the directory name as fzf's prompt to indicate the search is limited to that directory
         set --prepend fzf_arguments --prompt="Search Directory $unescaped_exp_token> " --preview="_fzf_preview_file $expanded_token{}"
-        set file_paths_selected $unescaped_exp_token(fd $fd_opts 2>/dev/null | _fzf_wrapper $fzf_arguments)
+        set file_paths_selected $unescaped_exp_token(command fd $fd_opts 2>/dev/null | _fzf_wrapper $fzf_arguments)
```

Comment:
Hi Susensio, unfortunately we should not always use the unwrapped version of fd because some users might actually aliased it to overcome an issue on Linux where fd is installed as fdfind. https://github.com/PatrickF1/fzf.fish/discussions/93